src/index.js

```diff
@@ -0,0 +1,141 @@
+const state = {
+    temp: 72, 
+    city: "Seattle", 
+    lat: 47.6038321, 
+    long: -122.3300624
+};
+
+
+const proxyLocation = "http://127.0.0.1:5000/location";
+const proxyWeather = "http://127.0.0.1:5000/weather";
+
+ const convertKtoF = (temp) => {
+     return (temp - 273.15) * (9/5) + 32;
+ };
+
+const findCoordinates = () => {
+    axios.get(proxyLocation, {params: {
+        q: state.city, 
+    }})
+    .then((response) => {
+        state.lat = response.data[0].lat;
+        state.long = response.data[0].lon;
+        getWeather();
+    })
+    .catch((error) => {
+        console.log('error in findCoordinates!', error.response);
+      });
+};
+
+const getWeather = () => {
+    axios.get(proxyWeather, {params: {
+        lat: state.lat,
+        lon: state.long
+    }})
+    .then((response) => {
+        const weather = response.data;
+        state.temp = Math.round(convertKtoF(weather.main.temp));
+        updateTempLandscape();
+    })
+    .catch((error) => {
+        console.log("Error getting the weather:", error);
+    });
+};
+
+const increaseTemp = () => {
+    state.temp += 1;
+    updateTempLandscape();
+};
+
+const decreaseTemp = () => {
+    state.temp -= 1;
+    updateTempLandscape();
+};
+
+const updateTempLandscape = () => {
+    let temp = state.temp;
+    let color = "above80";
+    let landscapeEmojis = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
```

Comment:
```suggestion
    let color;
    let landscapeEmojis;
```
Since you're just declaring these variables for later use, I'd recommend not assigning them content so that it's clear they are being prepped to receive values later.